mcp_server_gravitino/server/tools/models.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2024 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+
+from typing import Any, List, Optional, Tuple
+
+import httpx
+from fastmcp import FastMCP
+
+from mcp_server_gravitino.server.tools import metalake_name as global_metalake_name
+
+
+def get_list_of_models(mcp: FastMCP, session: httpx.Client) -> None:
+    """List all models in the given namespace"""
+
+    @mcp.tool(
+        name="get_list_of_models",
+        description="List all models in the given namespace.",
+        tags={
+            "model",
+            "list operation",
+        },
+    )
+    def _get_list_of_models(namespace: str) -> list[dict[str, Any]]:
```

Comment:
To be consistent with the "get_list_of_tables" tool, I suggest to use "catalog" and "schema" as the input paramters, instead of "namespace". Because in natual language communication, "catalog" and "schema" is more used way. Please refer to this: https://github.com/datastrato/mcp-server-gravitino/blob/main/mcp_server_gravitino/server/tools/table.py#L16